app/lib/primer/forms/acts_as_component.rb

```diff
@@ -66,30 +66,45 @@ def compile!
         @compiled = true
       end
 
-      private
-
       def template_root_path
         return @template_root_path if defined?(@template_root_path)
 
         form_path = Utils.const_source_location(self.name)
-        @template_root_path = if form_path
-          File.join(File.dirname(form_path), self.name.demodulize.underscore)
+        @template_root_path =
+          if form_path
+            File.join(File.dirname(form_path), self.name.demodulize.underscore)
+          end
+      end
+
+      def base_template_path
+        return @base_template_path if defined?(@base_path)
```

Comment:
Shouldn't it be `if defined?(@base_template_path)` instead of `if defined?(@base_path)`?
`@base_path` is never defined.

Reply:
Woops! Good catch, thanks 👍